From: Michal Hocko <mhocko@suse.com>
Subject: pc-mm-reduce-noise-in-show_mem-for-lowmem-allocations-update-fix-2
Date: Wed, 31 Aug 2022 09:58:35 +0200

fix build


lib/show_mem.c: In function '__show_mem':
lib/show_mem.c:17:9: error: too many arguments to function 'show_free_areas'
   17 |         show_free_areas(filter, nodemask, max_zone_idx);
      |         ^~~~~~~~~~~~~~~
In file included from lib/show_mem.c:8:
./include/linux/mm.h:1842:28: note: declared here
 1842 | static void __maybe_unused show_free_areas(unsigned int flags, nodemask_t *nodemask)

Link: https://lkml.kernel.org/r/Yw8Uq5rF19qVUQH+@dhcp22.suse.cz

Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/show_mem.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/show_mem.c~pc-mm-reduce-noise-in-show_mem-for-lowmem-allocations-update-fix-2
+++ a/lib/show_mem.c
@@ -14,7 +14,7 @@ void __show_mem(unsigned int filter, nod
 	unsigned long total = 0, reserved = 0, highmem = 0;
 
 	printk("Mem-Info:\n");
-	show_free_areas(filter, nodemask, max_zone_idx);
+	__show_free_areas(filter, nodemask, max_zone_idx);
 
 	for_each_online_pgdat(pgdat) {
 		int zoneid;
_
